native/src/components/RemoteContent.tsx

```diff
@@ -26,22 +25,22 @@ export const renderWebviewError = (
 
 type RemoteContentProps = {
   content: string
-  cacheDirectory: ParsedCacheDictionaryType
+  cacheDictionary: ParsedCacheDictionaryType
   language: string
   resourceCacheUrl: string
   onLinkPress: (url: string) => void
   onLoad: () => void
+  webViewWidth: number
 }
 
+// If the app crashes without an error message while using RemoteContent, consider wrapping it in a ScrollView or setting a manual height
```

Comment:
Is that a WIP comment or is it supposed to stay in? Hasn't crashed in my testing so far, btw.

Reply:
Its more for new usages of RemoteContent, I think I wrapped all current usages. Took me quite a while to figure out.